nodes/surface/interpolate_nurbs_surface.py

```diff
@@ -8,22 +8,27 @@
 from sverchok.data_structure import updateNode, zip_long_repeat, ensure_nesting_level, split_by_count
 from sverchok.utils.nurbs_common import SvNurbsMaths
 from sverchok.utils.surface.nurbs import SvGeomdlSurface, interpolate_nurbs_surface
+from sverchok.utils.surface.freecad import SvSolidFaceSurface
 from sverchok.utils.math import supported_metrics
-from sverchok.dependencies import geomdl
+from sverchok.dependencies import geomdl, FreeCAD
 
 if geomdl is not None:
     from geomdl import fitting
 
+if FreeCAD is not None:
+    import Part
+    from Part import BSplineSurface
 
-class SvExInterpolateNurbsSurfaceNode(SverchCustomTreeNode, bpy.types.Node):
+
+class SvExInterpolateNurbsSurfaceNodeMK2(SverchCustomTreeNode, bpy.types.Node):
     """
     Triggers: NURBS Surface Interpolate
     Tooltip: Interpolate NURBS Surface
     """
-    bl_idname = 'SvExInterpolateNurbsSurfaceNode'
+    bl_idname = 'SvExInterpolateNurbsSurfaceNodeMK2'
     bl_label = 'Interpolate NURBS Surface'
     bl_icon = 'SURFACE_NSURFACE'
-    sv_dependencies = {'geomdl'}
+    sv_dependencies = {'geomdl', 'FreeCAD'}
```

Comment:
@Durman says there is no need nowadays to include optional dependencies to `sv_dependencies`. So it's better to just remove this line.

Reply:
Done! Tested that the node works without that line.